lib/appoptics_apm/inst/em-http-request.rb

```diff
@@ -18,15 +18,12 @@ def setup_request_with_appoptics(*args, &block)
               report_kvs[:RemoteURL] = @uri
               report_kvs[:HTTPMethod] = args[0]
               report_kvs[:Blacklisted] = true if blacklisted
-
-              if AppOpticsAPM::Config[:em_http_request][:collect_backtraces]
-                report_kvs[:Backtrace] = AppOpticsAPM::API.backtrace
-              end
+              report_kvs[:Backtrace] = AppOpticsAPM::API.backtrace if AppOpticsAPM::Config[:em_http_request][:collect_backtraces]
```

Comment:
this is just moving to a consistent form?

Reply:
yes!
more readable for a rubyist. Nicer list of keys, followed by conditions.
Shorter method, still longer than the conventions recommend. Max. 7 lines.